.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.6.7
```

Comment:
The Ruby version in the `.ruby-version` file of that repo is 2.6.7 and the version in the Gemfile is 2.7.6. I assumed that the newer version in the Gemfile is the one you're using. Is that correct?